fmriprep/cli/parser.py

```diff
@@ -46,6 +47,8 @@ def _bids_filter(value):
         if value and Path(value).exists():
             return loads(Path(value).read_text())
 
+    _date_to_num = datetime.now().strftime('%Y%M%d')
```

Comment:
if this isn't random enough, we could always elect for a more "random" seed, aka
```python
random.choice(range(10000))
```

Reply:
I'm fine with the current date, as all we're looking for is reproducibility. If we do want to poll the current PRNG for something with more bits of entropy, I would suggest using `numpy.random.randint`. A bit more direct than `choice(range(n))`.